apps/studio/components/interfaces/Organization/BillingSettings/BillingBreakdown/UpcomingInvoice.tsx

```diff
@@ -146,12 +172,23 @@ const UpcomingInvoice = ({ slug }: UpcomingInvoiceProps) => {
                               />
                             }
                           />{' '}
-                          <span>
+                          <span className="mr-2">
                             {fee.description}
                             {fee.usage_metric &&
                               billingMetricUnit(fee.usage_metric) &&
                               ` (${billingMetricUnit(fee.usage_metric)})`}
                           </span>
+                          {(() => {
+                            const matchingTooltipData = feeTooltipData.find((it) =>
+                              fee.usage_metric?.startsWith(it.identifier)
+                            )
+                            return matchingTooltipData ? (
+                              <InvoiceTooltip
+                                text={matchingTooltipData.text}
+                                linkRef={matchingTooltipData.linkRef}
+                              ></InvoiceTooltip>
```

Comment:
Unless the component has children, we usually go for self-closing tags so <InvoiceTooltip .... />